src/data-units/DescriptorData.cpp

```diff
@@ -70,7 +70,7 @@ void *DescriptorData<T>::GetRequest() {
     return this->mpRequest;
 }
 
-#ifdef EXAGEOSTAT_USE_HICMA
+#ifdef USE_HICMA
```

Comment:
Will we use `USE_HICMA` instead of `EXAGEOSTAT_USE_HICMA` when building?

Reply:
Indeed, we have eliminated all instances of "EXAGEOSTAT_" in the CMake variables. No need to prolong them unnecessarily.